brian2/core/base.py

```diff
@@ -308,26 +304,20 @@ class BrianObjectException(Exception):
     original_exception : Exception
         The original exception that was raised.
     '''
-    def __init__(self, message, brianobj, original_exception):
+    def __init__(self, message, brianobj):
         self._brian_message = message
         self._brian_objname = brianobj.name
-        self._brian_origexc = '\n'.join(traceback.format_exception_only(type(original_exception),
-                                                                        original_exception))
-        self._brian_origtb = traceback.format_exc()
         self._brian_objcreate = brianobj._creation_stack
         logger.diagnostic('Error was encountered with object "{objname}":\n{fullstack}'.format(
             objname=self._brian_objname,
             fullstack=brianobj._full_creation_stack))
 
     def __str__(self):
-        return ('Original error and traceback:\n{origtb}\n'
-                'Error encountered with object named "{objname}".\n'
+        return ('Error encountered with object named "{objname}".\n'
                 '{objcreate}\n\n'
-                '{message} {origexc}'
+                '{message}'
                 '(See above for original error message and traceback.)'
-                ).format(origtb=self._brian_origtb,
-                         origexc=self._brian_origexc,
-                         objname=self._brian_objname, message=self._brian_message,
+                ).format(objname=self._brian_objname, message=self._brian_message,
```

Comment:
The spacing is not handled correctly in here, and leads to ``An error occurred when preparing an object.(See above for original error message and traceback.)``. Maybe just add a space before the ``(See above for``?